app/services/MatcherPool.scala

```diff
@@ -136,9 +136,9 @@ class MatcherPool(
     matchers.clear
   }
 
-  def getCurrentCategories: List[(String, Category, Int)] = {
+  def getCurrentCategories: List[(String, Set[Category], Int)] = {
```

Comment:
This is begging to be renamed and return a `List[NewBettererType]`

Reply:
Agreed. Hm, maybe we could just refactor in this PR. I'll give it a go.

Reply:
Done.